action.yml

```diff
@@ -69,9 +74,22 @@ runs:
     - name: 'Set target os'
       shell: bash
       run: echo "TARGET_OS=$(go env GOOS)" >> $GITHUB_ENV
+
+    - name: 'Set GoReleaser config file'
+      id: set_config
+      shell: bash
+      run: |
+        if [ -z "${{ inputs.goreleaser-config }}" ]; then
+          echo "No GoReleaser config file provided, using default"
+          echo "::set-output name=goreleaser_config::${{ github.action_path }}/.goreleaser.template.yml"
```

Comment:
AFAIK pattern:
```bash
echo "{name}={value}" >> "$GITHUB_OUTPUT"
```
is a preferred way for setting output value
See:
https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions#setting-an-output-parameter